packages/api/cms-api/src/translation/azure-ai-translator.resolver.ts

```diff
@@ -0,0 +1,47 @@
+import createClient, { isUnexpected, TextTranslationClient } from "@azure-rest/ai-translation-text";
+import { Inject } from "@nestjs/common";
+import { Args, Query, Resolver } from "@nestjs/graphql";
+
+import { RequiredPermission } from "../user-permissions/decorators/required-permission.decorator";
+import { AzureAITranslatorConfig } from "./azure-ai-translator.config";
+import { AZURE_AI_TRANSLATOR_CONFIG } from "./azure-ai-translator.constants";
+import { TranslationInput } from "./dto/translation.input";
+
+@Resolver()
+@RequiredPermission(["translation"], { skipScopeCheck: true })
+export class AzureAITranslatorResolver {
```

Comment:
It should Ai everywhere (except for CONSTANTS)

Reply:
a53e44842567e4c67db107a79418e3ae95139072